src/components/input-mask/input-mask.ts

```diff
@@ -7,19 +7,23 @@ import WithRender from './input-mask.html';
 
 // TODO : Hack en attendant qu'on puisse remettre cleaveOptions.
 export interface InternalCleaveOptions {
+    numericOnly?: boolean;
+    delimiters?: ReadonlyArray<string>;
```

Comment:
Depuis la version 3.4 de TS, tu peux utiliser `readonly string[]` C'est plus propre :)

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#a-new-syntax-for-readonlyarray

Reply:
J'ai essayé mais je me suis rendu compte qu'on était encore en 3.2 👀